auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationContextConfigurationClient.java

```diff
@@ -230,7 +234,6 @@ public SSLContext getSSLContext(URI uri, AuthenticationContext authenticationCon
      * @return the matching SSL context factory (not {@code null})
      */
     public SecurityFactory<SSLContext> getSSLContextFactory(URI uri, AuthenticationContext authenticationContext, String abstractType, String abstractTypeAuthority) {
-        Assert.checkNotNullParam("uri", uri);
```

Comment:
The javadoc for the parameter might need to be updated if it mentions that `null` is not allowed.

Reply:
Fixed